content/ekscloudwatchcontainerinsights/installwordpress.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Install Wordpress"
+chapter: false
+weight: 2
+---
+
+<h3>Install Wordpress to EKS Cluster</h3>
+  
+<img src="/images/ekscwci/wordpresslogo.png">
```

Comment:
We shouldn't need the embedded html... can you swap to markdown format, and also move images under /static/cwci/ (or some other relevant directory name)?